setup.py

```diff
@@ -98,7 +97,7 @@
         "Topic :: Database :: Database Engines/Servers",
         "Topic :: Database :: Front-Ends",
     ],
-    python_requires=">=3.8,<3.11",
+    python_requires=">=3.8,<3.12",
```

Comment:
```suggestion
    python_requires=">=3.8",
```

I think we should actually open this requirement up completely; this upgrade was fairly painless, beyond pinning one particular sub-dep, and as we are a hybrid library/application we don't want to the hold up for other packages!